pkg/kubelet/cm/cgroup_manager_linux.go

```diff
@@ -492,10 +492,7 @@ func setResourcesV2(cgroupConfig *libcontainerconfigs.Cgroup) error {
 	if err != nil {
 		return fmt.Errorf("failed to create cgroup v2 manager: %v", err)
 	}
-	config := &libcontainerconfigs.Config{
-		Cgroups: cgroupConfig,
-	}
-	return manager.Set(config)
+	return manager.Set(cgroupConfig.Resources)
```

Comment:
this can be `manager.Set(nil)` as the resources were passed to NewManager already. But it doesn't matter much.